collect_app/src/main/java/org/odk/collect/android/dao/helpers/InstancesDaoHelper.java

```diff
@@ -69,19 +66,6 @@ public static boolean isInstanceComplete(boolean end, boolean completedByDefault
         return complete;
     }
 
-    // TODO: replace with method in {@link org.odk.collect.android.instances.InstancesRepository}
-    // that returns an {@link Instance} object from a path.
-    public static Uri getLastInstanceUri(String path) {
```

Comment:
It looks like the repository method was added, so there wasn't really a need for this helper any longer.